### ISSUE TITLE
Visualisation spec

### ISSUE THREAD

**Issue by adamnieman** [carbonvisuals/eve#2]
Is the visualisation specification I added to the readme useful? How can I improve it?

**Comment by martinjoiner** [comment]
Yes the prototype graphic mockups are helpful. The key thing I need is all the "TODO"s. I have no idea what format ev_type might take for example.

**Comment by adamnieman** [comment]
Just checking (because you only mentioned the graphics) - you saw the visualisation specification at the bottom of the readme too, right?
